src/components/hosts/HostStatus.tsx

```diff
@@ -113,10 +116,11 @@ const HostStatus: React.FC<HostStatusProps> = ({ host }) => {
     <>
       <Popover
         headerContent={<div>{title}</div>}
-        bodyContent={getPopoverContent(host)}
+        bodyContent={getPopoverContent({ host, cluster, onValidationActionToggle })}
         footerContent={<small>Status updated at {getHumanizedDateTime(statusUpdatedAt)}</small>}
         minWidth="30rem"
         maxWidth="50rem"
+        hideOnOutsideClick={!keepOnOutsideClick}
```

Comment:
I'd expect the popover to hide when the host editing modal opens. Keeping the popover open causes the popover to overlay the modal and having 2 overlays open is quite confusing

Reply:
In that case I will need to move the modal component up in the hierarchy which is breaking the structure, imo.

If we keep the modal component down in the hierarchy (as it is now), an user's click inside of the modal closes the popover and subsequently destroys the modal component.

Or we might do bigger refactoring of (all) modals we have, shift all of them to a single higher place and drive their state via functions from context or redux. If we decide to go this way, we should implement it in a follow-up.